build.rs

```diff
@@ -114,14 +114,12 @@ fn main() {
     let mut build = cc::Build::new();
     build
         .file("./deps/ada.cpp")
-        .include("./deps/ada.h")
-        .include("./deps/ada_c.h")
+        .include("./deps")
```

Comment:
This would also include `ada.cpp` and `wasi_to_unknown.cpp`. Can we revert these changes?

Reply:
> This would also include `ada.cpp` and `wasi_to_unknown.cpp`. Can we revert these changes?

No, include specifies an _include path_ (cf [docs](https://docs.rs/cc/latest/cc/struct.Build.html#method.include)), so `ada.cpp` and `wasi_to_unknown.cpp` would not be included unless included explicitely via an `#include` directive.

Reply:
Technically, I don't think the include call is even needed, since the preprocessor should be able to resolve relative paths on its own. I'm not sure about the wasm target though.